gluten-ut/spark34/src/test/scala/org/apache/spark/sql/GlutenCachedTableSuite.scala

```diff
@@ -33,7 +33,7 @@ class GlutenCachedTableSuite
     super.sparkConf.set(GlutenConfig.COLUMNAR_TABLE_CACHE_ENABLED.key, "true")
   }
 
-  test("GLUTEN - InMemoryRelation statistics") {
+  testGluten("InMemoryRelation statistics") {
```

Comment:
@zwangsheng Just curious that, are you using IDEs? It occurs to be that Intellij Idea's scala tester doesn't resolve `testGluten` cases?

Reply:
Yes, it couldn't be triggered by the button, but maven tests were fine